cmk/base/legacy_checks/emerson_temp.py

```diff
@@ -40,7 +40,7 @@ def check_emerson_temp(item, params, info):
 
 
 def parse_emerson_temp(string_table: StringTable) -> StringTable:
-    return string_table
+    return [[x] for x in string_table[0]]
```

Comment:
There are many ways of doing this, obviously, but this way a test is failing that ensures we can deal with an emtpy input. I suggest to leave line 52 alone, add a comment, and do this:
```
def parse_...
    # we only use at most the first two lines because....
    return string_table[:2]
```